pkg/api/admin/openshiftcluster_example.go

```diff
@@ -0,0 +1,59 @@
+package admin
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func exampleOpenShiftCluster() *OpenShiftCluster {
+	return &OpenShiftCluster{
+		ID:       "/subscriptions/subscriptionId/resourceGroups/resourceGroup/providers/Microsoft.RedHatOpenShift/OpenShiftClusters/resourceName",
+		Name:     "resourceName",
+		Type:     "Microsoft.RedHatOpenShift/OpenShiftClusters",
+		Location: "location",
+		Tags: map[string]string{
+			"key": "value",
+		},
+		Properties: Properties{
+			ProvisioningState: ProvisioningStateSucceeded,
+			ClusterProfile: ClusterProfile{
+				Domain:          "cluster.location.aroapp.io",
+				Version:         "4.3.0",
+				ResourceGroupID: "/subscriptions/subscriptionId/resourceGroups/clusterResourceGroup",
+			},
+			ConsoleProfile: ConsoleProfile{
+				URL: "https://console-openshift-console.apps.cluster.location.aroapp.io/",
+			},
+			ServicePrincipalProfile: ServicePrincipalProfile{
+				ClientID: "clientId",
+			},
+			NetworkProfile: NetworkProfile{
+				PodCIDR:     "10.128.0.0/14",
+				ServiceCIDR: "172.30.0.0/16",
+			},
+			MasterProfile: MasterProfile{
+				VMSize:   VMSizeStandardD8sV3,
+				SubnetID: "/subscriptions/subscriptionId/resourceGroups/vnetResourceGroup/providers/Microsoft.Network/virtualNetworks/vnet/subnets/master",
+			},
+			WorkerProfiles: []WorkerProfile{
+				{
+					Name:       "worker",
+					VMSize:     VMSizeStandardD2sV3,
+					DiskSizeGB: 128,
+					SubnetID:   "/subscriptions/subscriptionId/resourceGroups/vnetResourceGroup/providers/Microsoft.Network/virtualNetworks/vnet/subnets/worker",
+					Count:      3,
+				},
+			},
+			APIServerProfile: APIServerProfile{
+				Visibility: VisibilityPublic,
+				URL:        "https://api.cluster.location.aroapp.io:6443/",
+				IP:         "1.2.3.4",
+			},
+			IngressProfiles: []IngressProfile{
```

Comment:
Would we expect to see Install, StorageSuffix populated in this struct as well?

Reply:
@jim-minter I'll add `StorageSuffix`, but not `Install`, I think. This example is in `ProvisioningState == ProvisioningStateSucceeded` state where we don't have `Install`.

Reply:
what to do sort of depends on what our static validation + test strategy is